app/javascript/Search/Search.jsx

```diff
@@ -14,33 +9,23 @@ const GLOBAL_SEARCH_KEY = 'Slash';
 const ENTER_KEY = 'Enter';
 
 export class Search extends Component {
-  static defaultProps = {
-    searchBoxId: 'nav-search',
-  };
-
   constructor(props) {
     super(props);
     this.enableSearchPageChecker = true;
     this.syncSearchUrlWithInput = this.syncSearchUrlWithInput.bind(this);
+    this.searchInputRef = createRef(null);
   }
 
   componentWillMount() {
-    let searchTerm;
+    const { searchTerm, setSearchTerm } = this.props;
```

Comment:
These are props now, because a hook in `<SyncSearchForms />` is what keeps the state between search forms synchronized.